src/applications/vaos/tests/new-appointment/components/DateTimeRequestPage.unit.spec.jsx

```diff
@@ -362,7 +362,7 @@ describe('VAOS <DateTimeRequestPage>', () => {
     });
   });
 
-  it.skip('should not allow selections after max date', async () => {
+  it('should not allow selections after max date', async () => {
```

Comment:
I haven't found an example of this test failing, just a comment in a PR, so I'm not sure what to try to fix. I'm just re-enabling it for now so we can track any future failures.